internal/dag/executor/docker.go

```diff
@@ -109,16 +125,6 @@ func (e *docker) Run() error {
 		}
 	}
 
-	out, err := cli.ContainerLogs(
-		ctx, resp.ID, types.ContainerLogsOptions{ShowStdout: true},
-	)
```

Comment:
Moved this above `cli.ContainerWait` so that log streaming starts immediately after container starts and not after the container exits